Dockerfile

```diff
@@ -7,7 +7,5 @@ RUN npm install --silent --no-cache
 
 COPY ./ ./
 
-RUN sh ./tests/fix_puppeteer.sh
```

Comment:
I'm sure I commented about this somewhere, but I cannot find it. Running tests upon build is quite useful, as it will break the build if they do not succeed